milestone2/admin-account-management-component.html

```diff
@@ -0,0 +1,256 @@
+            background-color: #2E4057;
+        }
+
+        .color-orange {
+            background-color: #E27D60;
+        }
+
+        .color-lightblue {
+            background-color: #97caef;
+        }
+
+        .color-lightorange {
+            background-color: #E8A87C;
+        }
+
+        .color-pink {
+            background-color: #C38D9E;
+        }
+
+        .color-green {
+            background-color: #41B3A3;
+        }
+
+        .color-corp-lightblue {
+            background-color: #265077;
+        }
+
+        .color-corp-darkblue {
+            background-color: #022140;
+        }
+
+        .color-corp-pink {
+            background-color: #494B68;
+        }
+
+        .color-corp-darkgreen {
+            background-color: #1E4258;
+        }
+
+        .color-corp-lightgreen {
+            background-color: #2D5F5D
+        }
+
+        .bg-color-lively-darkgreen {
+            background-color: #026670;
+        }
+
+        .bg-color-lively-lightgreen {
+            background-color: #9FEDD7;
+        }
+
+        .bg-color-lively-tan {
+            background-color: #FEF9C7;
+        }
+
+        .bg-color-lively-yellow {
+            background-color: #FCE181;
+        }
+
+        .bg-color-lively-grey {
+            background-color: #EDEAE5;
+        }
+
+        .text-color-lively-darkgreen {
+            color: #026670;
+        }
+
+        .text-color-lively-lightgreen {
+            color: #9FEDD7;
+        }
+
+        .text-color-lively-tan {
+            color: #FEF9C7;
+        }
+
+        .text-color-lively-yellow {
+            color: #FCE181;
+        }
+
+        .text-color-lively-grey {
+            color: #EDEAE5;
+        }
+
+        .text-color-jet {
+            color: #2F2D2E;
+        }
+
+        .min-width-1000 {
+            min-width: 1000;
+        }
+
+        .max-width-100 {
+            max-width: 100%;
+        }
+
+
+    </style>
+</head>
+<body>
+<div class="container-fluid color-lively-grey shadow min-width-1000 max-width-100">
+    <!-- Header -->
+    <div class="row">
+        <div class="col-8 col-sm-8 col-md-8 col-lg-8 col-xl-8 w-75">
+            <!-- Page Title -->
+            <div class="navbar-header m-2">
+                <a href="./index.html" class="navbar-brand ml-5"><h1 class="text-color-jet">Tri<b>angular</b> Arbitrary</h1></a>
+            </div>
+        </div>
+
+        <!-- User Greeting - TODO: add interpolation for userName {{ userName }} -->
+        <div class="col-4 col-sm-4 col-md-4 col-lg-4 col-xl-4 w-25" style="float: right;">
+            <div class="text-color-lively-darkgreen float-right m-3">
+                <span>Welcome, </span>
+                <a href="">John Doe</a>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="fluid-container md-form m-5 h-75 w-90 align-self-center">
+    <h1><b>Account Administration</b></h1>
+    <hr>
+
+    <div class="row">
+        <div class="col" style="width: 40%">
```

Comment:
Consider a min-width style for smaller screen resolutions that stops the elements from overlapping each other at those smaller resolutions